netstandardVersion.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
I think we should use netstandard2.0, any reason to use 2.1?

Reply:
Lookup this table https://github.com/dotnet/standard/blob/master/docs/versions.md, netstanard 2.1 if we are using .Net Core 3.1 implementation.

Reply:
I thought netstandard 2.0 wouldn't work due to build errors, but these were actually because I erroneously switched netcoreapp projects to netstandard. Fixed.